docs/relying-parties/tutorials/integrating-with-fxa.md

```diff
@@ -74,31 +78,37 @@ Before starting integration, please send a request to fxa-staff[at]mozilla.com t
 
 We communicate with our relying parties via the [firefox-accounts-notices group][firefox-accounts-notices].  Please join this list to avoid any surprises.
 
-## OAuth Integration
+## OpenID Connect Integration
+
+### Relying Party vs. Resource Server
+
+A Relying Party (RP) is an application or website that outsources its user authentication functionality to an OpenID Provider, defined as part of the OpenID specifications. The user will be prompted to login to the Relying Party with their Firefox Account, and is able to see the login as a distinct [Connected Service][connected-services] in FxA Settings. OpenID Connect contains many specifications for a variety of identity exchange and authorization, based on the OAuth 2.0 framework of specifications.
+
+In contrast, a Resource Server is an OAuth 2.0 term for an API server. A Relying Party may also act as a Resource Server, with a user never seeing a direct Relying Party login. Users that access a resource server with OAuth tokens will not see their access as a distinct Relying Party login on the FxA Settings page. Services that are accessed via Firefox (Sync, Relay, etc.) using an existing OAuth token are all examples of Resource Servers. Their access shows up in FxA Settings as the browser session the user is logged into.
 
 ### Development
 
 :::note
 You are encouraged to use [our staging servers](https://accounts.stage.mozaws.net/) to develop against.  Our staging server has a persistent database and changes made there are saved.  Spot testing and some new accounts are fine but you are expected to clean up any significant amount of data (eg. accounts made from automated testing).  If you're using PyFxA [here is an example](https://pypi.org/project/PyFxA/#testing-email-addresses) using `destroy_account()`.
 :::
 
-0. Review the [OAuth 2.0][oauth] documentation.
+0. Review the [OpenID Connect][how connect works] and [OAuth 2.0][oauth] documentation.
 0. Register for staging OAuth credentials by filing a [deployment bug][deployment-bug]. See [OAuth credentials](#oauth-credentials).
 0. Your development servers should point to: `https://oauth.stage.mozaws.net`.
-0. User authentication follows the [OAuth 2.0][oauth] protocol.
+0. User authentication follows the [OpenID Connect][openidconnect] protocol.
```

Comment:
Treating FxA as an OAuth provider should still work.  Or do we simply want to discourage that?

Reply:
OpenID Connect is built on OAuth, so using the higher level term still encompasses OAuth.